x-pack/plugins/fleet/server/services/epm/elasticsearch/template/template.ts

```diff
@@ -206,6 +206,7 @@ function _generateMappings(
           case 'boolean':
             dynProperties = {
               type: field.object_type,
+              time_series_metric: field.metric_type,
```

Comment:
Found that `metric_type` can directly used here, so not needed to put `time_series_metric` on the field.